src/main/java/org/junit/runner/Request.java

```diff
@@ -131,8 +131,11 @@ public Request filterWith(Filter filter) {
     }
 
     /**
-     * Returns a Request that only runs contains tests whose {@link Description}
-     * equals <code>desiredDescription</code>
+     * Returns a Request that only runs tests whose {@link Description}
+     * matches <code>desiredDescription</code>
+     *
+     * Returns an empty Request if <code>desiredDescription</code> is not a single test and filters all but the single
```

Comment:
The thing is this doesnt seem to work on the Intellij IDEA quick documentation view, where are you checking this changes? or how do you verify the links are displayed?